.circleci/config.yml

```diff
@@ -283,30 +367,80 @@ jobs:
                       yarn rollup -c test/visual/rollup.config.js
             - run:
                   when: always
-                  name: Publish review site
+                  name: Deploy Visual Tests to Azure Blob Storage
                   command: |
-                      cp projects/documentation/content/favicon.ico test/visual
-                      branch=$(git symbolic-ref --short HEAD)
-                      hash=$(echo -n $branch-hcm | md5sum | cut -c 1-32)
-                      echo "Netlify alias hash: $hash"
-                      echo "Deploying from test/visual directory..."
-
-                      max_attempts=3
-                      attempt=0
-
-                      while [ $attempt -lt $max_attempts ]; do
-                          if yarn netlify deploy --alias=$hash --cwd test/visual; then
-                              echo "Deployment successful!"
-                              exit 0
+                      # Get PR number from CircleCI environment
+                      set +x
+                      PR_NUMBER=""
+                      if [ -n "$CIRCLE_PULL_REQUEST" ]; then
+                          PR_NUMBER=$(echo $CIRCLE_PULL_REQUEST | sed 's/.*\/pull\///')
```

Comment:
Under what circumstance does this not exist? `CIRCLE_PR_NUMBER` shows to [always be there](https://circleci.com/docs/variables/#built-in-environment-variables). 🤔 
I'd love to get rid of the complexity below 😛